tests/src/Model/GServerTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
Think of adding the license header here.

Reply:
Done.